src/context.ts

```diff
@@ -33,10 +33,10 @@ export const MagicalContext = createContext<MagicalContext>({
 
 export const withWizardContext = <WrappedComponentProps = any>(
   Component: React.ComponentType<WrappedComponentProps>,
-) => {
-  const wizardProps = useContext(MagicalContext);
+): Function => {
+  const wizardProps: MagicalContext = useContext(MagicalContext);
```

Comment:
I also don't think that you need to specify type here, it's inferred

Reply:
✅